frontend/pages/dashboard/settings/profile.tsx

```diff
@@ -2,35 +2,64 @@ import React from 'react'
 import { NextPage } from 'next'
 import { withApollo } from '../../../lib/apollo'
 import SettingsPageLayout from '../../../components/Layouts/SettingsPageLayout'
-import Form from '../../../components/Form'
+import SettingsForm from '../../../components/Dashboard/Settings/SettingsForm'
+import SettingsFieldset from '../../../components/Dashboard/Settings/SettingsFieldset'
 import { useTranslation } from '../../../config/i18n'
-import Button, { Variant } from '../../../elements/Button'
+import Button, { ButtonVariant } from '../../../elements/Button'
 
 const ProfileInfo: NextPage = () => {
   const { t } = useTranslation('settings')
-  const loading = false
 
-  const handleSubmit = () => {}
+  const handleDetailsSubmit = (): void => {}
+  const handleLanguagesSubmit = (): void => {}
 
   return (
     <SettingsPageLayout>
-      <Form onSubmit={handleSubmit}>
-        <fieldset disabled={loading} aria-busy={loading}>
-          <legend>
-            <h2>{t('profileForm.legend')}</h2>
-          </legend>
-
-          <div className="details-wrapper">
-            <img className="profile-image" src="/images/robin-small.png" />
-
-            <Button className="details-submit-button" variant={Variant.Secondary}>
-              {t('profileForm.submitButton')}
-            </Button>
-          </div>
-        </fieldset>
-      </Form>
+      <div className="forms-container">
+        <SettingsForm onSubmit={handleDetailsSubmit}>
```

Comment:
What do you think of this structure? Since there will be about 5 sections/forms on the profile settings page, each with their own `Update`/submit button, I split them into multiple forms. It's probably possible to make one form technically, and then have multiple submit buttons within the single form, but this will depend on the Apollo/mutation setup. I'm open to structuring it both ways (1 form, multiple submit buttons / multiple forms, one submit button for each). I'm kind of thinking it makes sense to just start like this and if it makes sense to change it later depending on the Apollo stuff, I can make that change pretty quickly

Reply:
I'm down with starting like this and seeing if we need the change later 😄